cfme/tests/test_rest.py

```diff
@@ -191,11 +191,22 @@ def test_query_simple_collections(appliance, collection_name):
         casecomponent: Rest
         caseimportance: high
         initialEstimate: 1/3h
+        testSteps:
+            1. Send a GET request: /api/<collection_name>
+        expectedResults:
+            1. Must receive a 200 OK response.
     """
     collection = getattr(appliance.rest_api.collections, collection_name)
-    assert_response(appliance)
-    collection.reload()
-    list(collection)
+
+    if collection_name in COLLECTIONS_OMITTED:
+        # the "settings" and automate_workspaces collections are untypical
+        # as they don't have "resources" and for this reason can't be reloaded (bug in api client)
```

Comment:
Q: we have Bugzilla for it? If yes, why we are blocking?

Reply:
The bug is in the manageiq_client library, might not be a bug, but the feature has probably not been implemented in the library.

Collections not in `COLLECTION_OMITTED` can be easily accessed with `appliance.rest_api.collections.<collection_name>.all` since there is a `resource` attribute in the response, but we cannot access omitted collections such as `settings` and `automate_workspaces` as easily, since there is no `resource` attribute in the response.

Test this: `GET /api/providers` and `GET /api/automate_workspaces`, observe both the responses and look for `resources` attribute.

Reply:
@valaparthvi I think better to suggestion from devel for this.